### DISCUSSION TITLE
Is the self-hosted version limited in any way and if so how?

### DISCUSSION THREAD

**Discussion by Mugane** [AntlerVC/firetable#1442] [Support / Q&A] [answered]
Before spending a bunch of time to discover critical limitations of the self-hosted open-source version, is there somewhere where they are listed? If I have to install this in order to determine the limitations, and I discover any, the bad taste of that wasted time will ensure I never pay for or recommend the subscriptions, no matter how good the product is.

**Comment by harinij** [comment, marked as answer]
Hi @Mugane - thanks for reaching out. Our free cloud hosted version offers pretty much the same as the self hosted version - you can read the details here: [https://www.rowy.io/pricing](https://www.rowy.io/pricing)

Mainly, self hosted supports only one Firestore project on which you can create any number of tables connected to your collections. Plus some of the managed services like inviting users to the project etc is not available. Note there might be some intricacies with self hosting related to having a backend service installed as well - mainly if you want to use cloud function column types like derivations and actions. Finally self hosting might bring custom complexities depending on how your project / rules are setup which we might not be able to support / help debug.

If you want to give it a quick spin without much time invested you can use the free hosted version or checkout the [demo.rowy.io](https://demo.rowy.io)
